### PR TITLE
[CSP] Populate sandbox from MV2 field and log MV3 policy

### PR DESCRIPTION
**Pull request by solomonkinard**
https://developer.chrome.com/docs/extensions/mv3/intro/mv3-migration/#content-security-policy has more information on CSP for MV3.

### CODE REVIEW

File: src/modifiers/ContentSecurityPolicyModifier.py [hunk 1]
```diff
@@ -17,14 +17,17 @@ def _mv2(self):
...
-    value = manifest[key]
-    candidate = {
-      "extension_pages": value,
-      "sandbox": ""
-    }
+    if key in manifest:
+      value = manifest[key]
+      candidate["extension_pages"] = value
+    if "sandbox" in manifest and key in manifest["sandbox"]:
+      candidate["sandbox"] = manifest["sandbox"]
```
Comment (dotproto) [inline src/modifiers/ContentSecurityPolicyModifier.py]:
```suggestion
      candidate["sandbox"] = manifest["sandbox"][key]
```

I may be missing something, but it looks like the current impl is assigning the entire `sandbox` object from the original manifest to the new manifest's `content_security_policy.sandbox`. Also my Python is weak, so I'm not 100% sure if this is the right access pattern.

Comment (solomonkinard) [reply src/modifiers/ContentSecurityPolicyModifier.py]: Good catch.

Comment (solomonkinard) [reply src/modifiers/ContentSecurityPolicyModifier.py]: Good catch!

**Comment by solomonkinard** [comment]
Addressed comments. Added sandbox key deletion. Ignore OS files.

**Comment by solomonkinard** [comment]
Addressed comments. Added sandbox key deletion. Ignore OS files.

**Pull request merged**